ElitismHelper.lua

```diff
@@ -445,6 +444,21 @@ SlashCmdList["ELITISMHELPER"] = function(msg,editBox)
 			print(" activeUser: Prints active user")
 			print(" output: Define output channel between default | party | raid | yell | self")
 		end,
+		["threshold"] = function()
+			if argsFunc ~= nil then
+				thresholdNumber = tonumber(argsFunc, 10)
+				if thresholdNumber == nil then
+					print("Error: Threshold value not valid" .. argsFunc)
```

Comment:
Should exit early when invalid value is passed. Could be worth validating input? (ie, > 0 & <100)
```suggestion
					print("Error: Threshold value not valid " .. argsFunc)
					return
```

Reply:
Yep, added a specific check for 1-100 and added breaks to both validations